modules/express/src/clientRoutes.ts

```diff
@@ -1025,11 +1026,16 @@ export function createCustomSigningFunction(externalSignerUrl: string): CustomSi
     txPrebuild: TransactionPrebuild;
     pubs?: string[];
   }): Promise<SignedTransaction> {
-    const { body: signedTx } = await superagent
-      .post(`${externalSignerUrl}/api/v2/${params.coin.getChain()}/sign`)
-      .type('json')
-      .send({ txPrebuild: params.txPrebuild, pubs: params.pubs });
-
+    const { body: signedTx } = await retryPromise(
+      () =>
+        superagent
+          .post(`${externalSignerUrl}/api/v2/${params.coin.getChain()}/sign`)
+          .type('json')
+          .send({ txPrebuild: params.txPrebuild, pubs: params.pubs }),
+      (err, tryCount) => {
+        console.error(`attempt number ${tryCount}`);
```

Comment:
let's make this a debug instead and make it a little more informative
```suggestion
        debug(`failed to connect to external signer (attempt ${tryCount}, error: ${err.message})`);
```

Reply:
added to https://github.com/BitGo/BitGoJS/pull/1989